infrastructure/main.tf

```diff
@@ -42,10 +42,12 @@ module "payment-database" {
   env = var.env
   postgresql_user = var.postgresql_user
   database_name = var.database_name
-  sku_name = "GP_Gen5_2"
+  sku_name = var.sku_name
```

Comment:
Do we have any specific reason that these 2 lines are added only in payment-app? I couldn't see this change in other repos as part of DB upgrade?